test/unit/core/detail/type_inspection_test.cpp

```diff
@@ -0,0 +1,54 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include <seqan3/core/type_list/traits.hpp>
+#include <seqan3/core/type_list/type_list.hpp>
+#include <seqan3/core/detail/type_inspection.hpp>
+
+using namespace seqan3;
```

Comment:
We do not want to do this anymore

Reply:
since when, and what would be the rational for this?

Reply:
Please read up on the meeting notes if you haven't been there :wink:

Reply:
i did but i must have missed this.